toolchain/lowering/testdata/function/call/i32.carbon

```diff
@@ -0,0 +1,33 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// CHECK:STDOUT: ; ModuleID = 'i32.carbon'
+// CHECK:STDOUT: source_filename = "i32.carbon"
+// CHECK:STDOUT:
+// CHECK:STDOUT: %EmptyTupleType = type {}
+// CHECK:STDOUT:
+// CHECK:STDOUT: define i32 @Echo(i32 %a) {
+// CHECK:STDOUT: entry:
+// CHECK:STDOUT:   ret i32 1
+// CHECK:STDOUT: }
+// CHECK:STDOUT:
+// CHECK:STDOUT: define %EmptyTupleType @Main() {
+// CHECK:STDOUT: entry:
+// CHECK:STDOUT:   %var = alloca i32, align 4
+// CHECK:STDOUT:   %Echo = call i32 @Echo(i32 1)
+// CHECK:STDOUT:   store i32 %Echo, ptr %var, align 4
+// CHECK:STDOUT: }
+// CHECK:STDOUT:
+// CHECK:STDOUT: ; uselistorder directives
+// CHECK:STDOUT: uselistorder i32 1, { 1, 2, 0 }
```

Comment:
To answer the question in the description -- this is used to provide deterministic ordering of the use-def linked lists in the in-memory IR. We should suppress this in tests to make them less brittle I suspect. But that's probably best separated. I'll go digging for the API...

More details: https://llvm.org/docs/LangRef.html#use-list-order-directives

Reply:
Thanks -- for now, maybe it's okay to leave it.